numba_dpex/core/passes/parfor_lowering_pass.py

```diff
@@ -42,24 +50,16 @@ class ParforLowerImpl:
     for a parfor and submits it to a queue.
     """
 
-    def _submit_parfor_kernel(
-        self,
-        lowerer,
-        kernel_fn,
-        loop_ranges,
-    ):
-        """
-        Adds the call to the dpex kernel function from the main function.
-        """
+    def _prepare_queue_num_args(self, kernel_fn, lowerer):
```

Comment:
Can move the queue creation out to this function and rename the function to `_build_kernel_arglist`.